method/group.js

```diff
@@ -151,13 +150,34 @@ module.exports = function GroupMethods (ssb, keystore, state) {
             ),
             pull.map(msg => msg.value.content.excludes),
             pull.flatten(),
-            pull.unique(),
             pull.collect((err, excludedMembers) => {
               if (err) return cb(err)
 
-              // NOTE: this currently prevents people who've been removed from being re-added
-              // https://github.com/ssbc/ssb-tribes/issues/79
-              const members = addedMembers.filter(addedMember => !excludedMembers.includes(addedMember))
+              // calculate if the person has been added more times than they've been removed
+              // this is kind of basic but works. although lol people can add themselves back
```

Comment:
:fire: this crude version won't work sorry D:

The problem is if I add you to a Batts Group at the same time as Staltz adds you, then you've technically been added "twice", even though that addition was idempotent. Then I remove you ... then you've been added twice and removed once... so according to this you're in the group, even though you may have honoured the exclusion.

Reply:
mm ok i based this solution on a discussion we had at some point.

a simple solution could be to check this algorithm also whenever we receive something in the mutateMembers stream? to have the 2 systems match up, avoiding mismatches.

Reply:
let's try the tangle stuff in a separate pr instead https://github.com/ssbc/ssb-tribes/pull/95